### PR TITLE
Cowen

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [x] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

#1013. A tiny list of emotive concepts. I added a couple of new concept sets that are going to be linked to some other lists in concepticon in the next PR.

@LinguList would you be so kind to have a quick look at it if you have a minute?

### CODE REVIEW

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -353,3 +353,4 @@ Zalizniak-2020-2591	Zalizniak, Anna A. and Smirnitskaya, Anna and Russo, Maksim
...
+Cowen-2020-16	Cowen, Alan S. and Keltner, Dacher and Schroff, Florian and Jou, Brendan and Adam, Hartwig and Prasad, Gautam	2020		16	specific	English	Arabic, English, French, German, Indonesian, Italian, Japanese, Korean, Mandarin, Polish, Portuguese, Russian, Spanish, Thai, Turkish, Vietnamese	https://doi.org/10.1038/s41586-020-3037-7	Cowen2020		This list of 16 emotions emerged as a result of a machine-learning analysis of large-scaled video material from 144 countries. The purpose of the study was to find out what facial expressions are associated with various naturalistic social contexts and determine culturally universal patterns of context–expression association.
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv:356]: If all these languages are mentioned as "target languages", we need them also in our concept list. If the data cannot be found in the supplement anywhere, the additional languages should be discarded.

**Review by LinguList — COMMENTED** [review]
Thanks a lot for taking care of it. The only objection I have is the comment I mentioned.

I also wonder if we should consider adding also the other missing concepts?

File: concepticondata/conceptlists.tsv (hunk 1 above)
Comment (Kristina-Pianykh) [reply concepticondata/conceptlists.tsv:356]: As far as I understood, the list itself is made only in English while the material analyzed was derived from the videos in the languages listed. But I can see now that they should be removed from here. Shall I then just indicate 'global' in the 'target languages'?

Comment (Kristina-Pianykh) [reply concepticondata/conceptlists.tsv:356]: And I'll add the rest of the concept sets missing here.

**Comment by LinguList** [comment]
I was confused by teh columns: if it is not "source language", there won't be an error, so "target languages" as "global" is probably best, as this is a supposedly universal test.

**Review by LinguList — APPROVED** [review]

**Pull request merged**